apps/healthlocker/config/prod.secrets.exs

```diff
@@ -0,0 +1,21 @@
+use Mix.Config
```

Comment:
I'm not sure if this file is still use as it is not anymore imported int the config prod